frontend/vite-project/src/components/shipping-form/ShippingForm.tsx

```diff
@@ -0,0 +1,106 @@
+import { useShippingStore } from "../../stores/shippingStore";
+import './shippingForm.css'
+
+const ShippingForm = () => {
+  const { formData, updateFormField } = useShippingStore();
```

Comment:
```suggestion
  const formData = useFormData();
  const {updateFormField} = useShippingActions()
```
I'm sure you did this before I realized my mistake. Leaving it as it was would probably be fine for now, but if we added another piece of state, it could lead to issues. I just want to encourage good habits by using custom hooks for accessing state.